packages/suite/src/views/wallet/send/components/Outputs/components/Address/index.tsx

```diff
@@ -47,10 +48,32 @@ interface AddressProps {
     output: Partial<Output>;
 }
 
+const useLastTwoValues = () => {
+    const [lastTwoValues, setLastTwoValues] = useState<string[]>([]);
+
+    const addValue = (value: string) => {
+        let newValues;
+        setLastTwoValues(prevValues => {
+            newValues = [...prevValues, value];
+            if (newValues.length > 2) {
+                newValues.shift();
+            }
+
+            return newValues;
+        });
+
+        return newValues;
+    };
+
+    return { lastTwoValues, addValue };
+};
+
 export const Address = ({ output, outputId, outputsCount }: AddressProps) => {
     const [addressDeprecatedUrl, setAddressDeprecatedUrl] = useState<
         AddressDeprecatedUrl | undefined
     >(undefined);
+    type AddressChecksumInfo = { text: string } | undefined;
+    const [addressChecksumInfo, setAddressChecksumInfo] = useState<AddressChecksumInfo>(undefined);
```

Comment:
Why wrapping `text` into object?
`type` should not be inside component
```suggestion
const [addressChecksumInfo, setAddressChecksumInfo] = useState<string | undefined>();
```

Reply:
I realized this text is not used, it will be a boolean